Doc/tutorial/controlflow.rst

```diff
@@ -139,17 +139,16 @@ but in fact it isn't. It is an object which returns the successive items of
 the desired sequence when you iterate over it, but it doesn't really make
 the list, thus saving space.
 
-We say such an object is *iterable*, that is, suitable as a target for
+We say such an object is :term:`iterable`, that is, suitable as a target for
 functions and constructs that expect something from which they can
 obtain successive items until the supply is exhausted. We have seen that
-the :keyword:`for` statement is such an *iterator*. The function :func:`list`
-is another; it creates lists from iterables::
+the :keyword:`for` statement is such a construct, while an example of function
+that takes an iterable is :func:`sum`:
```

Comment:
Thanks @berkerpeksag , I have a question. Why you do not want the example to be covered by doctest? After writing the examples, I always run sphinx-build doctest against the file, in order to check if everything is ok.